setup.cfg

```diff
@@ -22,6 +22,9 @@ exclude_lines =
     if __name__ == .__main__.:
     add_model_specific_args
 
+[coverage:run]
+omit =
+    flash/image/classification/integrations/labelstudio/handler.py
```

Comment:
you explained something around this being it took too long for the tests to run, is this still the case or have I misunderstood @tchaton?

Reply:
I didn't check it out. I need to test it out.